easybuild/tools/filetools.py

```diff
@@ -97,7 +119,8 @@
     md5_class = hashlib.md5
     sha1_class = hashlib.sha1
 except ImportError:
-    import md5, sha
+    import md5
+    import sha
```

Comment:
we've moved on the Python 2.6 now, so `hashlib` will always be there, please update accordingly (no more fallback to `md5` and `sha`